content/20.treatments.md

```diff
@@ -302,8 +302,19 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+It was recently discussed [@doi:10.1093/jac/dkaa114] that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could also be administered to COVID-19 patients.
```

Comment:
```suggestion
As recently discussed [@doi:10.1093/jac/dkaa114], because the 90% effective concentration (EC~90~) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it may also be possible to achieve the effective concentration in COVID-19 patients.
```